WeakAuras/Types.lua

```diff
@@ -184,33 +189,49 @@ WeakAuras.unit_threat_situation_types = {
 
 WeakAuras.class_types = {}
 WeakAuras.class_color_types = {} -- TODO: it should be removed together with Bufftrigger (unused)
-for classID = 1, GetNumClasses() do
+for classID = 1, 20 do
```

Comment:
? Is GetNumClasses() removed in classic?

Reply:
Yes https://i.imgur.com/JIkuBtJ.png

Reply:
Ok, thanks. I'm still not totally sold on the magic 20 here, perhaps leave a comment identifying that for what it is?

Reply:
comment added